### DISCUSSION TITLE
First batch of boards available / Beta testers

### DISCUSSION THREAD

**Discussion by thomic** [dokuhn/openDTU-BreakoutBoard#23] [Show and tell]
_ENGLISH VERSION_
( _german below_ )
### Testing round / Boot Up
The first round of assembled PCBs (10 pieces) reached our desk. We tested them as far as we can, but as we see openDTU as a community project, we are looking for you as beta tester.
Our final goal is to create a community-based hardware which will be sold in a shop - this shop will donate most of its profits (deducting taxes and administration fees) to community projects (like hackerspaces) that help(ed) to bring this hardware project forward.

### How to become a beta tester? / What we expect?
We want to give out the first batch of boards to people, who actively help developing the community and knowledge around openDTU. As this is an open hardware project, we believe in the power of community - and this is topics we are looking for:
- Some experience with using Git and GitHub
- Willing to actively write wiki articles in this GitHub repo on how to setup and install openDTU board
- Willing to test openDTU board against different inverters and publish results as a wiki article in this GitHub repo
- Knowledge of HF engineering/testing - ability to test the board in several HF environments - come up with a test report
- Knowledge of electrical engineering/testing - ability to review and test the PCB - come up with a test report and come up with PullRequests or ideas for changes to the board
- Come up with your contribution idea for this repo

If you carefully read the above text, please write a mail with your short introduction, your field of experience and how you can support us, to 
betatest(AT)opendtu-board.de (replace (AT) with @-sign)
As we expect some mails incoming, we will decide who will become beta tester until 2023-04-30 (end of april).
You will get a mail from us to clarify shipping details afterwards.

### Second batch (mail if you are interested)
As we are planning to produce a second batch soon, to get more boards out in the community, we like to estimate how many boards we should order. We don't have a final pricing yet, but the final board will be around ~45 Euro per piece. This includes a share, which we will give back to community, including things like financing this beta test.

Please write a mail to
**secondbatch@opendtu-board.de**
with the number of boards you are interested in.

_GERMAN VERSION_
### Testrunde / Durchstarten
Die erste Runde der fertigen PCBs (10 Stück) hat uns erreicht. Wir haben diese soweit möglich getestet, aber wir möchten die Community als Beta-Tester mit einbinden. Deshalb suchen wir dich.
Unser Ziel ist es einen nicht-gewinnorientierten Shop aufzubauen, der seine Einnahmen (abzüglich Steuern und Verwaltungskosten) an die Community spendet - d.h. an Vereine und Organisationen die bei der (Weiter-)Entwicklung der Hard- und Software geholfen haben.

### Beta-Tester werden / Was wir erwarten?
Wir möchten die erste Produktionsrunde der Boards an Menschen versenden, die uns aktiv beim Aufbau einer Community und beim Wissenstransfer rund um openDTU helfen. Da wir glauben, dass die Community in open hardware und open source Projekten das wichtigste Gut ist, möchten wir euch aktiv in die Weiterentwicklung und Tests einbinden. Hier sind ein paar Themen, für die wir Unterstützung benötigen:
- Erfahrung im Umgang mit Git und GitHub
- Bereitschaft Wiki-Artikel über die Installation und Einrichtung des openDTU boards für dieses GitHub Repo zu schreiben
- Bereitschaft das openDTU board gegen verschiedene Mikrowechselrichter zu testen und die Ergebnisse im Wiki in diesem GitHub Repo zu veröffentlichen
- Wissen im Bereich Hochfrequenztechnik und die Möglichkeit das Board in verschiedenen HF Umgebungen zu testen - und einen Testbericht darüber zu schreiben
- Wissen im Bereich Elektrotechnik / Platinenentwicklung und die Möglichkeit die PCB zu überprüfen und testen - und einen Testbericht zu verfassen und entsprechende Änderungsvorschläge an uns zu senden - am besten per PullRequest
- Deine eigene Idee, wie du zu diesem Repository beitragen kannst

Wenn du den oben stehenden Text sorgfältig durchgelesen hast, schreibe bitte eine Mail mit einer kurzen Vorstellung von dir, deiner Expertise und wie du uns unterstützen kannst an 
betatest(AT)opendtu-board.de (ersetze (AT) with @-sign)
Da wir einigen Zulauf an Mails erwarten, werden wir über die Beta-Tester bis zum 2023-04-30 (Ende April) entscheiden.
Wir werden uns bei dir melden, um die Versanddetails zu klären.

### Zweite Runde (Schreibt wenn ihr Board(s) möchtet)
Da wir planen demnächst eine zweite Runde von Boards produzieren zu lassen, um noch mehr Boards in der Community zu verteilen, möchten wir gerne abfragen, wie groß das Interesse an diesem Board ist. Wir haben noch keinen finalen Preis, aber das fertige Board wird bei ca. 45 Euro pro Stück liegen. Darin enthalten ist bereits ein Anteil, den wir an die Community / Vereine zurückgeben möchten und mit dem sich z.B. auch die Boards für die Beta-Tests finanzieren.

Bitte schreibe eine Mail an
**secondbatch@opendtu-board.de**
mit der Anzahl der Boards, die du bestellen würdest.

**Comment by thomic** [comment]
Time shift
---
Hi all,
sorry due to sickness, this got delayed a bit. We will reach out to those who wrote us until next week CW20 / 2023.

Hallo Welt,
Entschuldigung, aber aufgrund einer krankheitsbedingten Pause, mussten wir den Termin verschieben, Wir werden uns bei allen, die uns geschrieben haben, bis nächste Woche KW20 / 2023 melden.

**Comment by andy5macht** [comment]
Hallo,
kannst du schon was sagen, wann die ersten Boards raus gehen? Habe bisher keine Mail bekommen.

**Comment by thomic** [comment]
Closing as this died while launching. Sad but true.
